src/schemas/map.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$id": "http://asl-validator.cloud/map#",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "Type": {
+      "type": "string",
+      "pattern": "^Map$"
+    },
+    "Next": {
+      "type": "string"
+    },
+    "End": {
+      "enum": [true]
+    },
+    "Comment": {
+      "type": "string"
+    },
+    "OutputPath": {
+      "type": ["string", "null"]
+    },
+    "InputPath": {
+      "type": ["string", "null"]
+    },
+    "ResultPath": {
+      "type": ["string", "null"]
+    },
+    "ItemsPath": {
+      "type": ["string", "null"]
+    },
+    "MaxConcurrency": {
+      "type": "number",
+      "minimum": 0
+    },    
+    "Iterator": {
+      "$ref": "http://asl-validator.cloud/state-machine#"
+    },
```

Comment:
I think we are missing the `Parameters` property which is kinda [hidden in the docs](https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-map-state.html#map-state-example-params), but its usage is confirmed in [ASL specs](https://states-language.net/spec.html#state-type-table):
![image](https://user-images.githubusercontent.com/1325474/65589815-fb3b6c80-df89-11e9-97f8-c340489c5a7b.png)

Reply:
well caught!

Reply:
added!